src/include/codegen/transaction_manager/transaction_manager.h

```diff
@@ -0,0 +1,80 @@
+//===----------------------------------------------------------------------===//
+//
+//                         Peloton
+//
+// transaction_manager.h
+//
+// Identification: src/include/codegen/transaction_manager/transaction_manager.h
+//
+// Copyright (c) 2015-16, Carnegie Mellon University Database Group
+//
+//===----------------------------------------------------------------------===//
+
+/**
+ * This file contains the C API of the transaction manager.
```

Comment:
Oh, these are created because the C++ calling convention is more complicated. We will try to call into C++ though, so these might well get deleted later.